src/snovault/elasticsearch/indexer.py

```diff
@@ -285,41 +458,124 @@ def index(request):
 
         if errors:
             result['errors'] = errors
 
-        if record:
+        if request.json.get('record', False):
             try:
-                es.index(index=INDEX, doc_type='meta', body=result, id='indexing')
+                request.registry[ELASTIC_SEARCH].index(
+                    index=request.registry.settings['snovault.elasticsearch.index'],
+                    doc_type='meta',
+                    body=result,
+                    id='indexing'
+                )
             except:
                 error_messages = copy.deepcopy(result['errors'])
                 del result['errors']
-                es.index(index=INDEX, doc_type='meta', body=result, id='indexing')
+                request.registry[ELASTIC_SEARCH].index(
+                    index=request.registry.settings['snovault.elasticsearch.index'],
+                    doc_type='meta',
+                    body=result,
+                    id='indexing'
+                )
                 for item in error_messages:
                     if 'error_message' in item:
                         log.error('Indexing error for {}, error message: {}'.format(item['uuid'], item['error_message']))
                         item['error_message'] = "Error occured during indexing, check the logs"
                 result['errors'] = error_messages
 
 
-        es.indices.refresh(RESOURCES_INDEX)
+        request.registry[ELASTIC_SEARCH].indices.refresh(RESOURCES_INDEX)
         if flush:
             try:
-                es.indices.flush_synced(index=RESOURCES_INDEX)  # Faster recovery on ES restart
+                request.registry[ELASTIC_SEARCH].indices.flush_synced(index=RESOURCES_INDEX)  # Faster recovery on ES restart
             except ConflictError:
                 pass
+        return result, indexing_update_infos
 
+    # Load indexing
+    output_tuple = _load_indexing(request, session, connection, indexer_state)
+    result, invalidated, flush, first_txn, snapshot_id, restart, xmin, return_now = output_tuple
+    if return_now:
+        return result
+    indexing_update_infos = []
+    dry_run = request.json.get('dry_run', False)
+    if invalidated and not dry_run:
+        run_indexing_vars = (
+            request,
+            indexer_state,
+            result,
+            invalidated,
+            stage_for_followup,
+            flush,
+            snapshot_id,
+            restart,
+            xmin,
+        )
+        if this_node:
+            if this_node['node_index'] == HEAD_NODE_INDEX:
+                # Check remote indexing
+                uuids_count = len(invalidated)
+                do_remote_indexing = _determine_indexing_protocol(request, uuids_count)
+                if did_timeout:
+                    log.warning(f"Remote indexing index: did_timeout.  Run on head node.")
+                    head_node, indexing_node, time_now, did_fail, is_indexing_node  = setup_indexing_nodes(
```

Comment:
multiple flake8 issues, here, none of the variables are ever used and there are 2 spaces before =